doc/release/2.4.1.rst

```diff
@@ -98,8 +110,9 @@ Misc
     real port when bound to the port ``0``. For example, if the
     ``listen`` parameter of ``box.cfg`` is set to ``127.0.0.1:0``, the
     ``box.info.listen`` shows ``127.0.0.1:<real_port>`` (gh-4620).
+    Read more: :doc:`/reference/reference_lua/box_info/box_info_listen`.
```

Comment:
this statement causes a warning. Should be:
```suggestion
    Read more: :doc:`/reference/reference_lua/box_info/listen`.
```

Reply:
Corrected.